.github/SECURITY.md

```diff
@@ -0,0 +1,32 @@
+## Reporting Vulnerabilities
+
+We’re extremely grateful for security researchers and users that report vulnerabilities to the Hasura GraphQL Community. All reports are thoroughly investigated by a set of community volunteers.
+
+To make a report, please email us at [build@hasura.io](mailto:build@hasura.io) with the complete security details attaching all necessary information.
+
+### When Should I Report a Vulnerability?
+
+- You think you discovered a potential security vulnerability in the Hasura GraphQL Engine or related components.
+- You are unsure how a vulnerability affects the Hasura GraphQL Engine.
+- You think you discovered a vulnerability in another project that Hasura GraphQL Engine depends on (e.g. heroku, docker, etc)
+- Any other risk that would harm the Hasura GraphQL Engine.
+
+### When Should I NOT Report a Vulnerability?
+
+- You need help tuning Hasura GraphQL Engine components for security.
+- You need help applying security related updates.
+- Your issue is not security related.
+
+## Security Vulnerability Response
+
+Each report is acknowledged and analyzed by Product Security Team members within 3 working days. 
+
+As the security issue moves from triage, to identified fix, to release planning we will keep the reporter updated.
```

Comment:
The reporter will be kept updated at every stage of the issue's analysis and resolution (triage -> fix -> release).